src/Ddeboer/DataImport/Filter/DateTimeThresholdFilter.php

```diff
@@ -47,14 +47,14 @@ public function __construct(
     /**
      * {@inheritDoc}
      */
-    public function filter(array $item)
+    public function __invoke(array $item)
```

Comment:
Not only this, in my opinion its easier to use for newbies and in my projects i do this `new CallbackFilter(function() { /* some code */ });`ever and ever again. So i thought its easier to use, but i know that this isn't a really good design decision. 

I already had a discussion with @sagikazarmark about that feature in this PR (see comments above). I with you have the same opinion as @sagikazarmark, i will revert them.